lua/lspsaga/finder/box.lua

```diff
@@ -173,7 +173,7 @@ function M.indent(ns, lbufnr, lwinid)
           virt_text_pos = 'overlay',
           ephemeral = true,
         })
-        api.nvim_set_hl(0, hi, { link = 'Comment', default = true })
+        api.nvim_set_hl(0, hi, { link = 'SagaVirtLine', default = true })
```

Comment:
there is for finder indent . so maybe it's better not changed here or can we give it `SagaIndent` ?